### PR TITLE
Code review pull request 

### PR DESCRIPTION
**Pull request by HussenYesufAlli**
In this project I:
      Add the header section 
     add welcome section
     add the project section

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @HussenYesufAlli , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a review project, where you did not make a review at all, there is only an open pull request. Please make a review on your project using github review features

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (HussenYesufAlli) [inline index.html:7]: Please give the descriptive title which is directly express your project page. The contents of a page title can have significant implications for search engine optimization

Comment (HussenYesufAlli) [inline index.html:7]: Please give the descriptive title which is directly express your project page. The contents of a page title can have significant implications for search engine optimization

File: style.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+/* .imgdiv img{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+    grid-gap: 4rem;
+    margin: 0 auto;
+      margin-bottom: 0px;
+    margin-bottom: 6rem;
+    width: 350px;
+    height: 300px;
+} */
```
Comment (HussenYesufAlli) [inline style.css:74]: Please do not leave unused code as a comment in your codebase. unused code as a comment makes your code not clean and bulky.

**Review by HussenYesufAlli — COMMENTED** [review]
# Changes Required ♻️ 

Good job so far!
Your project is looking good so far 🤩

However, there are some changes you need to address. Please check my comments below and make all required changes. 


Thanks 😁

File: index.html (hunk 1 above)
Comment (Shedrack-Sunday) [reply index.html:7]: - [ ] You did great in this section, however, please use a spell checker such as Grammarly to check for errors and sentences 👍 For example, you made a mistake in "which is directly express your project page."

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @HussenYesufAlli 

Good job so far! 👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- You need to take cognizance of the suggested changes asked such as the use of error free grammar. I've made a detailed review in this regard below 



## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (HussenYesufAlli) [reply index.html:7]: Please give a descriptive title that directly expresses your project page. The contents of a page title can have significant implications for search engine optimization.

**Review by LionRouge1 — APPROVED** [review]
Hi @HussenYesufAlli, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**